src/qibolab/platform.py

```diff
@@ -31,6 +60,16 @@ class Settings:
     relaxation_time: int = int(1e5)
     """Time in ns to wait for the qubit to relax to its ground state between shots."""
 
+    def apply(self, options: ExecutionParameters):
```

Comment:
`settings.apply(options)` doesn't sound like the correct operations. Maybe better something like `settings.fill(options)`.